L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -153,3 +153,44 @@ def customiseSimL1EmulatorForPostLS1_Additional_HI(process):
     if hasattr(process,'caloConfig'):
         process.caloConfig.fwVersionLayer2 = cms.uint32(1)
     return process
+
+# dustomization for offline DQM
+def customiseStage1ForOfflineDQM(process):
```

Comment:
where is this function called?
75X is not based on eras
this function call probably fits in postLS1Customs.customise_DQM_25ns (if it's appropriate for both data and MC)
maybe even in customise_DQM